docs/contributing.md

```diff
@@ -17,7 +17,7 @@
 
```

Comment:
All instances of `design-system-tailwind-preset` updated to `design-tokens`

![Screenshot 2025-01-28 at 2 38 22 PM](https://github.com/user-attachments/assets/319e92fd-929d-41de-a1ca-992fd75de458)